### ISSUE TITLE
Solve naming confusion

### ISSUE THREAD

**Issue by hanshoglund** [music-suite/music-score#332]
The core combinators for sequential/parallel composition in Music Suite are:
- Parallel composition: `<>` and `pcat`
- Sequential composition: `|>` and `scat`.

Currently parallel composition is the same as `mappend/mconcat`, and sequential composition is a defined in terms of parallel. This works great for types with position-style semantics such as `Score` as these types always compose in parallel, and the position can be used to define all other styles of composition (this is a direct copy of `hcat`, `vcat` etc in Diagrams).

However when using types with duration-style semantics such as `Note` and `Voice` the above approach is confusing. For these types `scat` is currently not defined and `pcat` means what `scat` _ought_ to mean, when `pcat` should really be undefined and `scat` be the same as mconcat.

We could solve this with an extra type class, but I'd rather not.

Compare Tiled PTM (Hudak et al), in which `%` is _both_ sequential and parallel (depending on the boundaries defined for the values in question). This is an intriguing idea, but it is unclear how to integrate this with the Diagrams vector/point semantics that we currently use.

Possibly related to #325.

**Comment by hanshoglund** [comment]
This repo has been merged into: https://github.com/music-suite/music-suite

Closing :)
